tests/test_workers/test_tasks/test_build_regenerate_bundle.py

```diff
@@ -115,7 +115,7 @@ def test_handle_regenerate_bundle_request(
         ]
     )
 
-    mock_capml.assert_called_once_with(request_id, list(arches))
+    mock_capml.assert_called_once_with(request_id, list(arches), [])
```

Comment:
I am missing tests for create_empty_index, merge_index_image

Reply:
because there weren't any which works with real _create_and_push_manifest_list

Reply:
oh, that's why. Thank you for explaining :)